### DISCUSSION TITLE
[Suggestions] Should we make it so that the colored rectangles that represent the resource generators on the minimap are displayed in full even if part of them is "hidden" by decors on the adventure map

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#6119] [General]
In fheroes2, should we make it so that the colored rectangles that represent the resource generators on the minimap are displayed in full even if part of them is "hidden" by decors on the adventure map.

In fheroes2:

https://user-images.githubusercontent.com/43583994/200328354-0d4e7154-b337-405b-8fab-cec0ba750bbb.mp4

**Comment by zenseii** [comment]
It is probably better to have them in full to avoid anyone misinterpreting what building they are representing. In fact it's always confused me that they were made up of different shapes.
